docs/en/05_Contributing/01_Code.md

```diff
@@ -86,6 +86,20 @@ If you are changing existing APIs, introducing new APIs or major new features,
 please review our guidance on [supported versions](release_process#supported-versions).
 You will need to change the branch for your pull request accordingly.
 
+SilverStripe public APIs explicitly include:
```

Comment:
Can we please limit this to PHP APIs? No explicit guarantees about JS APIs. We'll try to keep them stable, but the reality of wrangling a large CMS UI built over a decade is that there's too many moving parts for API stability - at least until we've got this mess under control again through a ReactJS refactor (where we'll create a different kind of mess over time heh)

Reply:
added JS and everything non-php to explicitly excluded